recipes/libtorrent/all/conanfile.py

```diff
@@ -97,18 +117,26 @@ def _patch_sources(self):
             tools.patch(**patch_data)
 
         tools.replace_in_file(os.path.join(self._source_subfolder, "CMakeLists.txt"), "/W4", "")
-        if self.options.enable_iconv:
-            replace = "find_public_dependency(Iconv REQUIRED)"
+        if tools.Version(self.version) < "2.0":
+            if self.options.enable_iconv:
+                replace = "find_public_dependency(Iconv REQUIRED)"
+            else:
+                replace = "set(Iconv_FOUND OFF)"
+            tools.replace_in_file(os.path.join(self._source_subfolder, "CMakeLists.txt"),
+                                  "find_public_dependency(Iconv)",
+                                  replace)
+            if self.settings.compiler == "clang" and self.settings.compiler.libcxx == "libstdc++":
+                # https://github.com/arvidn/libtorrent/issues/3557
+                tools.replace_in_file(os.path.join(self._source_subfolder, "include", "libtorrent", "file_storage.hpp"),
+                                      "file_entry& operator=(file_entry&&) & noexcept = default;",
+                                      "file_entry& operator=(file_entry&&) & = default;")
         else:
-            replace = "set(Iconv_FOUND OFF)"
-        tools.replace_in_file(os.path.join(self._source_subfolder, "CMakeLists.txt"),
-                              "find_public_dependency(Iconv)",
-                              replace)
-        if self.settings.compiler == "clang" and self.settings.compiler.libcxx == "libstdc++":
-            # https://github.com/arvidn/libtorrent/issues/3557
-            tools.replace_in_file(os.path.join(self._source_subfolder, "include", "libtorrent", "file_storage.hpp"),
-                                  "file_entry& operator=(file_entry&&) & noexcept = default;",
-                                  "file_entry& operator=(file_entry&&) & = default;")
+            min_compiler_version = {
+                "gcc": "5",
+            }.get(str(self.settings.compiler))
+            if min_compiler_version:
```

Comment:
Theres another min compiler check. Hard to tell on mobile but this seems like a duplicate

Reply:
I think you're right.